assets/src/js/search.js

```diff
@@ -39,6 +29,10 @@ renderResultList = function(resultName, limit) {
             html += '<li><p>No results.</p></li>';
         }
         html += '</ul>';
+    } else if (resultName === 'other' ){
+        html += '<ul>';
+        html += '<li>hit enter to view more search results.</li>';
```

Comment:
Can't ElasticSearch provide autocomplete results?

Reply:
it can but it needs to be implemented ;) did that for opensearch already so its not hard to add it here. will do that.

Reply:
done. 8a152ca